webpack.config.babel.js

```diff
@@ -260,27 +269,26 @@ const baseConfig = {
   devtool: PRODUCTION_BUILD ? 'source-map' : 'eval-cheap-source-map',
 }
 
-const configDist = {
+const configDist = () => ({
   ...baseConfig,
 
   entry: {
-    onfido: './index.tsx',
+    [`onfido${SDK_ENV === 'auth' ? SDK_ENV : ''}`]: './index.tsx',
```

Comment:
IMO it should be `'Auth'` rather than `'auth'` to form `onfidoAuth` instead of `onfidoauth`.

Reply:
We used lowercase to keep all characters consistent. Mostly to avoid any mismatching if the SDK_ENV variable is set with all lower case rather than capitalised. I can change it if you'd prefer? although files generated in `dist` are named `onfidoAuth` (with capital A)